libs/leonsans/src/leonsans.ts

```diff
@@ -0,0 +1,608 @@
+/*!
+ * VERSION: 1.6.3
+ * DATE: 2019-09-13
+ * https://leon-sans.com
+ *
+ * @license Copyright (c) 2019, Jongmin Kim. All rights reserved.
+ **/
+import { Graphics } from 'pixi.js';
+
+import { MAX_FONT_WEIGHT, MIN_FONT_WEIGHT } from './core/constants.js';
+import { Dispatcher } from './core/dispatcher';
+import { Model } from './core/model';
+import { shuffle } from './core/util';
+import { Color } from './draw/canvas/color.js';
+import { Colorful } from './draw/canvas/colorful.js';
+import { Grids } from './draw/canvas/grids.js';
+import { Lines } from './draw/canvas/lines.js';
+import { Pattern } from './draw/canvas/pattern.js';
+import { Points } from './draw/canvas/points.js';
+import { Wave } from './draw/canvas/wave.js';
+import { PixiColor } from './draw/pixi/color.js';
+import { PixiLines } from './draw/pixi/lines.js';
+
+type LeonSansProps = {
+  text?: string;
+  size?: number;
+  weight?: number;
+  color?: (string | string[])[];
+  colorful?: string[];
+  tracking?: number;
+  leading?: number;
+  align?: Align;
+  pathGap?: number;
+  amplitude?: number;
+  width?: number;
+  breakWord?: boolean;
+  fps?: number;
+  isPattern?: boolean;
+  isWave?: boolean;
+  hasGrid?: boolean;
+  hasPoint?: boolean;
+  hasBox?: boolean;
+};
+
+export default class LeonSans extends Dispatcher {
```

Comment:
```suggestion
/**
 * Leon Sans is a geometric sans-serif typeface made with code in 2019 by Jongmin Kim.
 * It allows to change font weight dynamically and to create custom animations, effects or shapes in the Canvas element of HTML5.
 */
export default class LeonSans extends Dispatcher {
```

사용자가 직접 접근하게 되는 constructor니까 좀 꾸며져 있으면 좋을 것 같아요! 설명은 깃헙에서 따왔음.